docs/getting-started.md

```diff
@@ -33,50 +33,47 @@ CREATE scully.config.js (65 bytes)
 UPDATE package.json (1507 bytes)
 ```
 
-#### IMPORTANT
-*Scully needs the router be present in your application, do not forget add it.*
-
 ## @scullyio/init:blog
 
+#### IMPORTANT: *Scully requires the router to be present in your application, don't forget to add it.*
+
```

Comment:
Im ok with the styling, but we want to keep the original position.

Reply:
Fixed! Thanks for checking the PR 😄 !